config.yaml

```diff
@@ -39,10 +39,10 @@
             # Make sure that the samples you merge have the same sample features (e.g paired, sense, dup_type)
             # Choose an informative experiment name
   experiment_name: {
-                    rbp: 'rbp name', #rna binding protein name, eg: PUM2,
-                    replicates: ["fg_sample1"], # sample(s) to be treated as foreground.
```

Comment:
See my previous comment. Should rather be 
# names you choose for sample(s) that should be treated as foreground